tagging/create_manifests.py

```diff
@@ -11,7 +11,7 @@
 from .manifests import ManifestHeader, ManifestInterface
 
 
-logger = logging.getLogger(__name__)
+LOGGER = logging.getLogger(__name__)
```

Comment:
Not sure what I think in general about either small case or upper case. Can you put your finger on why you want to have it upper case rather than lower case?

It is a change off topic to the PR, so I'm leaning towards thinking it would been better to avoided complicating this otherwise very extensive PR.

Reply:
I observe below that this makes things consistent within the project. Due to that I suggest we include this change and resolve this comment.